src/foam/box/HTTPBox.js

```diff
@@ -103,8 +103,8 @@ foam.CLASS({
         cls.extras.push(foam.java.Code.create({
           data: `
 protected class Outputter extends foam.lib.json.Outputter {
```

Comment:
Is there a reason foam.lib.json.Outputter isn't modelled? If it was, we could use the class property here instead of javaExtras.